.github/workflows/build_and_test.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
     needs: [build]
     strategy:
       matrix:
-        version: [ v1.23.6, v1.24.0, v1.25.3 ]
+        version: [ v1.24.0, v1.25.3, 1.26.0 ]
```

Comment:
s/1.26.0/v1.26.0/

Reply:
good catch, this is why CI is failing, can't find the kind docker image with the right tag